### DISCUSSION TITLE
希望能加一个隐藏动态话题的功能

### DISCUSSION THREAD

**Discussion by YZRHinory** [the1812/Bilibili-Evolved#4284] [功能建议 / Ideas]
b站动态的话题过于逆天了，明明该把这些引流的话题推送放到首页的。阿b就是要这么搞来恶心人，部分话题又逆天又低能，很难不让人觉得难受。

**Comment by Geeyun-JY3** [comment]
你说的是动态首页侧栏的话题列表？

**Comment by YZRHinory** [comment]
是的，动态右侧栏的话题

from：suyuyin
________________________________

**Comment by YZRHinory** [comment]
是的，动态右侧栏的话题版块
________________________________

**Reply by Geeyun-JY3** [reply]
用[`动态过滤器`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E5%8A%A8%E6%80%81%E8%BF%87%E6%BB%A4%E5%99%A8)啊

**Comment by YZRHinory** [comment]
收到，感谢❤️😘

Geeyun-JY3 ***@***.***> 于 2023年7月15日周六 23:21写道：
